### PR TITLE
Recipe for BeScreenCapture 2.2

### PR DESCRIPTION
**Pull request by jackburton79**
I just released a new version of BeScreenCapture and created a new recipe.
This is really just the old recipe with the new url for the new release.

### CODE REVIEW

File: haiku-apps/bescreencapture/bescreencapture-2.2.recipe [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+PROVIDES="
+	bescreencapture = $portVersion
+	app:BeScreenCapture = $portVersion
+	"
+PROVIDES_inputfilter="
+	bescreencapture_inputfilter = $portVersion
+	app:BeScreenCaptureInputFilter = $portVersion
+	"
+
+REQUIRES="
```
Comment (humdingerb) [inline haiku-apps/bescreencapture/bescreencapture-2.2.recipe]: I think normally, the provides/required blocks of the sub-packages are kept together (like the summary/description). So you ccould move this "required" up under its "provides".

File: haiku-apps/bescreencapture/bescreencapture-2.2.recipe [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+DESCRIPTION_inputfilter="Input Server Addon for BeScreenCapture. Allows the \
+user to launch BeScreenCapture and start/stop recording using a keyboard \
+combination (CTRL-COMMAND-SHIFT + R)."
+
+HOMEPAGE="https://github.com/jackburton79/bescreencapture/releases"
+COPYRIGHT="2014-2016 Stefano Ceccherini"
+LICENSE="BSD (3-clause)
+	MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/jackburton79/bescreencapture/archive/v2.2.tar.gz"
```
Comment (humdingerb) [inline haiku-apps/bescreencapture/bescreencapture-2.2.recipe]: you could use "v$portsVersion" here to safe changes for more versions in the future.

**Comment by jackburton79** [comment]
Il 09/ott/2016 18:08, "humdinger" <notifications@github.com> ha scritto:
>
="
>
> I think normally, the provides/required blocks of the sub-packages are
kept together (like the summary/description). So you ccould move this
"required" up under its "provides".
>

This, AFAIR, was pointed up also last time, but I think someone said this
version was fine, too.

**Comment by jackburton79** [comment]
Il 09/ott/2016 18:09, "humdinger" <notifications@github.com> ha scritto:
>
> @humdingerb commented on this pull request.

>
> you could use "v$portsVersion" here to safe changes for more versions in
the future.
>

You're the second person who told me that. Last time I said I preferred to
keep it like this, since I am not sure I'll krep using this tag scheme, but
I Guess that I'll just have to accept this, since this seems to be so
popular :-)

**Comment by jackburton79** [comment]
I'll close this pull request and create a new one, since I just released a bugfix release (2.2.1).
In this one I changed the things you pointed out.

**Comment by humdingerb** [comment]
WRT to using "v$portsVersion" in the SOURCE_URI, I think using it you can only win: If you keep the naming scheme you save editing one more line, if you don't, you'd have to edit the line anyway. :)
Sorry for the late reply.

**Pull request closed without merging**